ale/drivers/cassini_drivers.py

```diff
@@ -6,11 +6,11 @@
 import numpy as np
 
 from ale import config
-from ale.drivers.base import Framer, RadialDistortion, Driver, Pds3, NaifSpice
+from ale.drivers.base import Framer, RadialDistortion, Driver, Pds3Label, NaifSpice
 
-
-class CassiniIssPds3NaifSpiceDriver(Driver, Pds3, NaifSpice, Framer, RadialDistortion):
+class CassiniIssPds3NaifSpiceDriver(Driver, Pds3Label, NaifSpice, Framer, RadialDistortion):
```

Comment:
Super nitpicky, but the name should probably have the word "label", so:  `CassiniIssPds3LabelNaifSpiceDriver`. Just for the sake of consistency.